core/plannernotes.c

```diff
@@ -156,11 +156,11 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 		gaschange_before =  (gasmix_distance(&lastprintgasmix, &gasmix) || lastprintsetpoint != dp->setpoint);
 		/* do we want to skip this leg as it is devoid of anything useful? */
 		if (!dp->entered &&
-		    nextdp &&
-		    dp->depth.mm != lastdepth &&
-		    nextdp->depth.mm != dp->depth.mm &&
-		    !gaschange_before &&
-		    !gaschange_after)
+			nextdp &&
```

Comment:
i think this was the correct indent before.

edit: when its a `if (` line like that with multiple lines bellow it, you space align the multiple lines. in this case it had 4 spaces